persons.rb

```diff
@@ -1,8 +1,7 @@
-require './nameable'
 require './base_decorator'
-require './capitalize_decorator'
+require './nameable'
 require './trimmer_decorator'
-
+require './capitalize_decorator'
 
 class Person < Nameable
   def initialize(name, age = 'Unknown', parent_permission: true)
```

Comment:
- [ ] Please kindly ensure that there is a method for adding rentals
- [ ] Also, please ensure that getters and setters for @rentals, name, and age is present